src/Components/test/E2ETest/Tests/ThreadingAppHostedTest.cs

```diff
@@ -110,11 +110,4 @@ private void WaitUntilLoaded()
         var app = Browser.Exists(By.TagName("app"));
         Browser.NotEqual("Loading...", () => app.Text);
     }
-
-    public void Dispose()
-    {
-        // Make the tests run faster by navigating back to the home page when we are done
-        // If we don't, then the next test will reload the whole page before it starts
-        Browser.Exists(By.LinkText("Home")).Click();
```

Comment:
In the multithreading case, this happens too late and was trying to reach the DOM after the page is gone. Fortunately we just don't even need this optimization.